papermill/execute.py

```diff
@@ -233,15 +233,18 @@ def _translate_type_r(val):
     return _translate_escaped_str(val)
 
 _translate_type_str_scala = _translate_escaped_str
-_translate_type_int_scala = _translate_to_str
 _translate_type_float_scala = _translate_to_str
```

Comment:
Should we have a similar rule for floats vs. doubles?

Reply:
I guess that would be more of an optimization than fixing a limitation. Since doubles are the default & can cover the range that floats support.

Reply:
Yep. I think it's fine as is.